scripts/generate-google3-docs/resource-reference/generated/resource-docs/gkehub/gkehubfeaturemembership.md

```diff
@@ -1202,6 +1202,134 @@ spec:
+    management: MANAGEMENT_AUTOMATIC
+---
+apiVersion: container.cnrm.cloud.google.com/v1beta1
+kind: ContainerCluster
+metadata:
+  annotations:
+    cnrm.cloud.google.com/project-id: gkehubfeaturemembership-dep-csau
+  name: gkehubfeaturemembership-dep-csau
+spec:
+  location: us-central1-a
+  initialNodeCount: 3
+  workloadIdentityConfig:
+    # Workload Identity supports only a single namespace based on your project name.
+    workloadPool: gkehubfeaturemembership-dep-csau.svc.id.goog
+---
+apiVersion: gkehub.cnrm.cloud.google.com/v1beta1
+kind: GKEHubFeature
+metadata:
+  name: gkehubfeaturemembership-dep-csau
+  annotations:
+    cnrm.cloud.google.com/deletion-policy: abandon
+spec:
+  projectRef:
+    name: gkehubfeaturemembership-dep-csau
+  location: global
+  # The resourceID must be "configmanagement" if you want to use Anthos config
+  # management feature.
+  resourceID: configmanagement
+---
+apiVersion: gkehub.cnrm.cloud.google.com/v1beta1
+kind: GKEHubMembership
+metadata:
+  annotations:
+    cnrm.cloud.google.com/project-id: gkehubfeaturemembership-dep-csau
+  name: gkehubfeaturemembership-dep-csau
+spec:
+  location: us-central1
+  authority:
+    # Issuer must contain a link to a valid JWT issuer. Your ContainerCluster is one.
+    issuer: https://container.googleapis.com/v1/projects/gkehubfeaturemembership-dep-csau/locations/us-central1-a/clusters/gkehubfeaturemembership-dep-csau
+  description: A sample GKE Hub membership
+  endpoint:
+    gkeCluster:
+      resourceRef:
+        name: gkehubfeaturemembership-dep-csau
+---
+apiVersion: resourcemanager.cnrm.cloud.google.com/v1beta1
+kind: Project
+metadata:
+  name: gkehubfeaturemembership-dep-csau
+spec:
+  name: Config Connector Sample
+  organizationRef:
+    # Replace "${ORG_ID?}" with the numeric ID for your organization
+    external: "${ORG_ID?}"
+  billingAccountRef:
+    # Replace "${BILLING_ACCOUNT_ID?}" with the numeric ID for your billing account
+    external: "${BILLING_ACCOUNT_ID?}"
+---
+apiVersion: serviceusage.cnrm.cloud.google.com/v1beta1
+kind: Service
+metadata:
+  annotations:
+    cnrm.cloud.google.com/project-id: gkehubfeaturemembership-dep-csau
+    cnrm.cloud.google.com/deletion-policy: "abandon"
+  name: gkehubfeaturemembership-dep1-csau
+spec:
+  resourceID: container.googleapis.com
+---
+apiVersion: serviceusage.cnrm.cloud.google.com/v1beta1
+kind: Service
+metadata:
+  annotations:
+    cnrm.cloud.google.com/project-id: gkehubfeaturemembership-dep-csau
+    cnrm.cloud.google.com/deletion-policy: "abandon"
+  name: gkehubfeaturemembership-dep2-csau
+spec:
+  resourceID: gkehub.googleapis.com
+---
+apiVersion: serviceusage.cnrm.cloud.google.com/v1beta1
+kind: Service
+metadata:
+  annotations:
+    cnrm.cloud.google.com/project-id: gkehubfeaturemembership-dep-csau
+    cnrm.cloud.google.com/deletion-policy: "abandon"
+  name: gkehubfeaturemembership-dep3-csau
+spec:
+  resourceID: anthosconfigmanagement.googleapis.com
+```
+
```

Comment:
@ziyue-101 , this seems almost the same as https://cloud.google.com/config-connector/docs/reference/resource-docs/gkehub/gkehubfeature#anthos_config_management_feature. Can we avoid redefining these dependencies in every example?